config/charts/inferencepool/templates/_helpers.tpl

```diff
@@ -31,3 +31,22 @@ Selector labels
 {{- define "gateway-api-inference-extension.selectorLabels" -}}
 inferencepool: {{ include "gateway-api-inference-extension.name" . }}
 {{- end -}}
+
+
+{{/*
```

Comment:
Will there be any issue if these are not removed? Basically I think we should set the env vars based on the `trace` flag as part of the helm chart, and users using our helm shouldn't run into this inconsistent state.